lib/Common/Memory/HeapBucket.cpp

```diff
@@ -1651,6 +1663,9 @@ void
 HeapBucketGroup<TBlockAttributes>::SweepFinalizableObjects(RecyclerSweep& recyclerSweep)
 {
     finalizableHeapBucket.Sweep(recyclerSweep);
+#ifdef RECYCLER_VISITED_HOST
+    recyclerVisitedHostHeapBucket.Sweep(recyclerSweep);
```

Comment:
Curious- is there a reason that these need to be swept before the smallFinalizableWithBarrier heap bucket? Or was this just an arbitrary choice?

Reply:
I believe its arbitrary.  I think it was because Daniel first added the code without the RECYCLER_VISITED_HOST directives and it looked neater organizing the non-conditionally-compiled code together.